modules/videoio/src/cap_android_camera.cpp

```diff
@@ -732,6 +766,18 @@ class AndroidCameraCapture : public IVideoCapture
         cameraManager = nullptr;
         imageReader = nullptr;
     }
+
+    template<typename FuncT, typename T>
+    bool submitRequest(FuncT setFn, uint32_t tag, const T &data)
+    {
+        ACaptureRequest *request = captureRequest.get();
+
+        return request &&
+               setFn(request, tag, 1, &data) == ACAMERA_OK &&
+               ACameraCaptureSession_setRepeatingRequest(captureSession.get(),
```

Comment:
Why do we need it for every ACaptureRequest_setEntryXXX call?

Reply:
Yes. It is not sufficient to update the metadata only. Re-submitting the capture request will make the change effective after the current request has finished.